frontend/src/app/passwordReset/MfaSelect/MfaSelect.tsx

```diff
@@ -0,0 +1,119 @@
+import { useEffect, useState } from "react";
+
+import { Card } from "../../commonComponents/Card/Card";
+import { CardBackground } from "../../commonComponents/CardBackground/CardBackground";
+import TextInput from "../../commonComponents/TextInput";
+import Button from "../../commonComponents/Button/Button";
+import RadioGroup from "../../commonComponents/RadioGroup";
+import { emailIsValid } from "../../utils/email";
+
+export const MfaSelect = () => {
+  const [emailAddress, setEmailAddress] = useState("");
+  const [emailError, setEmailError] = useState("");
+  const [formIsDirty, setFormIsDirty] = useState(false);
+  const [mfaOption, setMfaOption] = useState("");
+  const [mfaOptionError, setMfaOptionError] = useState("");
+
+  const validateEmail = () => {
+    setFormIsDirty(true);
+    if (!emailAddress) {
+      setEmailError("Enter your email address");
+      return;
+    }
+    let valid;
+    try {
+      valid = emailIsValid(emailAddress);
+    } catch (e) {
+      valid = false;
+    }
```

Comment:
what's the reason to write like that? 
in any case, it will be false if it's not inputted. This should work: 
```suggestion
    const valid = emailIsValid(emailAddress);
```

Reply:
Seems like it wouldn't work from Nick's comment in his PR so resolving